### DISCUSSION TITLE
dhaqsoPay discussion | Tuesday, 12 November 2024, 8:00 PM

### DISCUSSION THREAD

**Discussion by sharafdin** [dugsiiyeinc/dhaqsoPay#2] [Zoom Links]
zoom link:
https://us06web.zoom.us/j/84590803734?pwd=b5Xsegm3BgfEgbDVTIFNg67PlBeMmO.1

**Comment by sharafdin** [comment]
@dugsiiyeinc/dhaqsopay-contributors soo gala team

**Comment by omartood** [comment]
### dugsiiye waa madasha open-sources-ka

**Comment by sharafdin** [comment]
Zoom ka waa uu inoo dhamaaday si waafina waa isku fahannay 😍 @dugsiiyeinc/dhaqsopay-contributors ❤️✊🏿

**Comment by A-M-Ahmed** [comment]
It was wonderful to gain more information during this meeting.
